webapp/src/components/AssetFilters/AssetFilters.container.ts

```diff
@@ -67,6 +69,9 @@ const mapState = (state: RootState, ownProps: OwnProps): MapStateProps => {
         ? values.maxEstateSize || ''
         : getMaxEstateSize(state),
     rarities: 'rarities' in values ? values.rarities || [] : getRarities(state),
+    status: ('status' in values
```

Comment:
the `status` value will have multiple selections?

```suggestion
    status: ('status' in values) ? values.status : getStatus(state) as AssetStatusFilter,
```